tests/.e2e-env

```diff
@@ -1,12 +1,9 @@
 POSTGRES_USER="postgres"
 POSTGRES_PASSWORD="postgres"
-POSTGRES_DB="data"
-POSTGRES_TABLE="medic"
+POSTGRES_DB="cht_sync"
+POSTGRES_TABLE="couchdb"
 POSTGRES_SCHEMA="v1"
-DBT_POSTGRES_USER="postgres"
-DBT_POSTGRES_PASSWORD="postgres"
-DBT_POSTGRES_SCHEMA="dbt"
-DBT_POSTGRES_HOST="postgres"
```

Comment:
yay! I see these are actually no longer used - this is great tech debt reduction!

Reply:
Yes, it should have been updated when the DBT environment variables were removed.